### PR TITLE
added RegisterPageSuccess

### PR DESCRIPTION
**Pull request by avvaland26**
is this looks ok?

### CODE REVIEW

**Comment by mital20** [comment]
yes it is good

**Comment by avvaland26** [comment]
ok

**Review by mital20 — COMMENTED** [review]
this looks  much better

**Review by avvaland26 — COMMENTED** [review]
ok

File: .idea/uiDesigner.xml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          <preferred-size width="150" height="-1" />
+        </default-constraints>
+      </item>
+      <item class="javax.swing.JPasswordField" icon="/com/intellij/uiDesigner/icons/passwordField.png" removable="false" auto-create-binding="true" can-attach-label="true">
+        <default-constraints vsize-policy="0" hsize-policy="6" anchor="8" fill="1">
+          <preferred-size width="150" height="-1" />
+        </default-constraints>
+      </item>
+      <item class="javax.swing.JFormattedTextField" icon="/com/intellij/uiDesigner/icons/formattedTextField.png" removable="false" auto-create-binding="true" can-attach-label="true">
+        <default-constraints vsize-policy="0" hsize-policy="6" anchor="8" fill="1">
```
Comment (mital20) [inline .idea/uiDesigner.xml:52]: this need to be changed

File: src/main/java/WebTest/BrowserManager.java [hunk 2]
```diff
@@ -7,6 +7,8 @@
 
 public class BrowserManager extends Util{
     public void openBrowser(){
+       LoadProp loadProp = new LoadProp();
+       String BrowserName = loadProp.getProperty("chrome");
```
Comment (mital20) [inline src/main/java/WebTest/BrowserManager.java:11]: change this to home page

Comment (avvaland26) [reply src/main/java/WebTest/BrowserManager.java:11]: ok

**Comment by avvaland26** [comment]
ok

File: src/main/java/WebTest/BrowserManager.java [hunk 3]
```diff
@@ -7,6 +7,8 @@
 
 public class BrowserManager extends Util{
     public void openBrowser(){
+       LoadProp loadProp = new LoadProp();
```
Comment (mital20) [inline src/main/java/WebTest/BrowserManager.java:10]: this not good change it now

**Pull request merged**